packages/block-editor/src/components/media-replace-flow/index.js

```diff
@@ -171,21 +122,22 @@ const MediaReplaceFlow = ( {
 								} }
 							/>
 						</MediaUploadCheck>
-						{ onSelectURL && (
-							<MenuItem
-								icon={ link }
-								onClick={ () =>
-									setShowURLInput( ! showURLInput )
-								}
-								aria-expanded={ showURLInput }
-							>
-								<div> { __( 'Insert from URL' ) } </div>
-							</MenuItem>
-						) }
 					</NavigableMenu>
-					{ showURLInput && (
+					{ onSelectURL && (
 						<div className="block-editor-media-flow__url-input">
-							{ urlInputUIContent }
+							<span className="block-editor-media-replace-flow__image-url-label">
+								{ __( ' Current media URL:' ) }
+							</span>
+							<LinkControl
+								value={ { url: mediaURLValue } }
+								settings={ false }
```

Comment:
It might work, but technically it's not of a valid type.

https://github.com/WordPress/gutenberg/tree/master/packages/block-editor/src/components/link-control#settings

I think this should be more correct for what you're trying to do:

```suggestion
								settings={ [] }
```